File: Python/3842.py
# Problem no 3842 - Leetcode 


# class Solution:
#     def toggleLightBulbs(self, bulbs):
#         freq = {}

#         # Count occurrences
#         for b in bulbs:
#             freq[b] = freq.get(b, 0) + 1

#         result = []

#         # Check odd occurrences
#         for bulbID, count in freq.items():
#             if count % 2 != 0:
#                 result.append(bulbID)

#         result.sort()
#         return result

File: C++/3842.cpp
// LeetCode Problem no 3842


// class Solution {
// public:
//     vector<int> toggleLightBulbs(vector<int>& bulbs) {
//         unordered_map<int, int>freq;

//         for(int b : bulbs){
//             freq[b]++;
//         }

//         vector<int> result;

//         for (auto const& [bulbID, count] : freq) {
//             if (count % 2 != 0) {
//                 result.push_back(bulbID);
//             }
//         }
//         sort(result.begin(), result.end());

//         return result;
//     }
// };